smartapps/smartthings/ecobee-connect.src/ecobee-connect.groovy

```diff
@@ -671,7 +788,8 @@ void controlSwitch( dni, desiredState ) {
 			try {
 				httpPut(params) { resp ->
 					keepTrying = false
-					log.debug "RESPONSE CODE: ${resp.status}"
+					def rspDataString = "${e.resp?.data}".toString()
```

Comment:
do we have access to `e` here? I think this will error out.

Reply:
copy/paste issue but the string is used in the log